kratos/modified_shape_functions/tetrahedra_3d_4_ausas_incised_shape_functions.cpp

```diff
@@ -0,0 +1,152 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Ruben Zorrilla
+//
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "modified_shape_functions/tetrahedra_3d_4_ausas_incised_shape_functions.h"
+
+namespace Kratos
+{
+
+/// Tetrahedra3D4AusasIncisedShapeFunctions implementation
+/// Default constructor
+Tetrahedra3D4AusasIncisedShapeFunctions::Tetrahedra3D4AusasIncisedShapeFunctions(const GeometryPointerType pInputGeometry,
+        const Vector& rNodalDistancesWithExrapolated, const Vector& rExtrapolatedEdgeRatios) :
+    Tetrahedra3D4AusasModifiedShapeFunctions(pInputGeometry, rNodalDistancesWithExrapolated), mExtraEdgeRatios(rExtrapolatedEdgeRatios) {};
+
+/// Destructor
+Tetrahedra3D4AusasIncisedShapeFunctions::~Tetrahedra3D4AusasIncisedShapeFunctions() {};
+
+/// Turn back information as a string.
+std::string Tetrahedra3D4AusasIncisedShapeFunctions::Info() const {
```

Comment:
```suggestion
std::string Tetrahedra3D4AusasIncisedShapeFunctions::Info() const
{
```
Formatting (same in the other methods)